lib/evmone/instructions_calls.cpp

```diff
@@ -82,20 +89,18 @@ evmc_status_code call_impl(StackTop stack, ExecutionState& state) noexcept
         state.gas_left += 2300;
     }
 
-    state.return_data.clear();
```

Comment:
This wasn't tested in the ethereum/tests?

Reply:
What do you mean by "tested"? This is non-functional change. In particular, the return buffer clear was moved up to be consistent with create.